tests/common/src/main/resources/SnappySqlPoolTestCode.txt

```diff
@@ -143,4 +143,38 @@ val expected2_3 = Seq(
 assert(rs.length == expected2_3.length)
 assert(rs.toSeq.sortBy(_.getLong(0)) == expected2_3)
 
+
+val JDBC_URL=s"jdbc:snappydata:pool://$host:$port"
+// Client Pool Driver Test - Start
+val connProps = new java.util.Properties()
+connProps.setProperty("pool-maxTotal", "50")
+connProps.setProperty("pool-maxIdle", "20")
+connProps.setProperty("pool-initialSize", "50")
+connProps.setProperty("pool-minActive", "50")
+connProps.setProperty("pool-minIdleSize", "10")
+connProps.setProperty("driver", "io.snappydata.jdbc.ClientPoolDriver")
+connProps.setProperty("pool-user", userName)
+connProps.setProperty("pool-password", password)
+
```

Comment:
@PradeepSurale Shouldn't these properties be "pool." rather than hyphen?

Reply:
@PradeepSurale Have made the change to "pool." and will push it with other fixes for occasional test failures